views/partials/footer.ejs

```diff
@@ -1,5 +1,5 @@
-<footer class="footer bg-dark py-3 mt-auto">
-  <div class="container">
-    <span class="text-muted">&copy;Pixelify</span>
+<footer>
+  <div class="myFooter">
```

Comment:
Why not put the class on the `footer` element and remove the `div` underneath?

Reply:
Think i ended up doing that because it didn't work the simpler way for some reason, then again its the 3-4am footer, might have just been my fault it didnt work.